### DISCUSSION TITLE
Mouse allows scrolling upward

### DISCUSSION THREAD

**Discussion by tsaraki** [helix-editor/helix#10512] [Troubleshooting]
### Summary

![image](https://github.com/helix-editor/helix/assets/21110604/8b1791aa-f5d7-4b97-af13-c9d0bf3b0bed)
![image](https://github.com/helix-editor/helix/assets/21110604/c19af395-c04f-483e-81cc-c3430d1c1241)

empty log

also there is no bufferline

### Reproduction Steps

 

I tried this:

1. `hx`
2. mouse wheel scroll up

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

win 11

### Terminal Emulator

wt 1.12.10983.0

### Installation Method

release page

### Helix Version

helix-term 24.3 (2cadec0b)

**Comment by pascalkuthe** [comment]
I have no idea what this issue is about these screenshots are not meaningful, provide a proper description.

**Comment by RoloEdits** [comment]
Hmm, looks like they are saying while in helix(In Windows terminal), they can scroll up, thus pushing the view down? (They mention bufferline, but the statusline as that would be "scrolled" out of view if scrolling up).

**Comment by gabydd** [comment]
this seems to be a problem with entering the alternate screen the only other place I have seen this is the unix tty I am not sure if we can do anything about this maybe try updating windows terminal 1.12 is around 2 years old?

**Comment by RoloEdits** [comment]
Using:
Windows Terminal: `1.19.10821.0` 
Helix:  Custom branch, but rebased up to `9df126637632`
Windows: Windows 11 Pro; 23H2; 22631.3447

I followed the provided reproduction steps of 
- `hx`
- scroll up

And failed to reproduce the result. 

Also tried opening a new tab and alternating between them and trying to scroll, and failed to reproduce the issue.

**Comment by tsaraki** [comment]
I cant reproduce this bug anymore, idk
